src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Controllers/ValuesController.fs

```diff
@@ -1,14 +1,11 @@
 ﻿namespace Company.WebApplication1.Controllers
 
-open System
```

Comment:
Why remove these?

Reply:
Unused

Reply:
We intentionally keep commonly used usings here. Let's undo this bit.

Reply:
ok